src/main/java/org/mskcc/cbio/oncokb/web/rest/UsageAnalysisController.java

```diff
@@ -127,4 +130,19 @@ private JSONObject requestData(String file)
 
         return new ResponseEntity<UsageSummary>(summary, status);
     }
+
+    @GetMapping("/usage/resource/detail")
```

Comment:
Should we just do `/user/resource/{endpoint}`?

Reply:
'/' in endpoint will cause matching error.

Reply:
But I believe the api request has encoded the slash?